cmd/ci_view.go

```diff
@@ -77,6 +78,11 @@ Feedback Encouraged!: https://github.com/zaquestion/lab/issues`,
 			log.Fatal(err)
 		}
 		projectID = project.ID
+		branch, err := lab.GetBranch(projectID, branchName)
```

Comment:
this needs redo as before `view [remote [branch/tag]]` was accepting both as ref not only  branchName

I am thinking of sth like 
```golang
func FindCommitByRef(pid interface{}, ref string) (*gitlab.Commit, error) {}
```
via `lab.Commits.ListCommits` https://docs.gitlab.com/ce/api/commits.html#list-commits

Reply:
done on https://github.com/zaquestion/lab/pull/380/commits/468bb6916394a58f32ee962b98cdbdbbefac920f